src/common/primitive_hashing.cpp

```diff
@@ -757,6 +768,13 @@ size_t get_desc_hash(const sdpa_desc_t &desc) {
     // Scale type
     seed = hash_combine(seed, static_cast<size_t>(desc.scale_dt));
     seed = hash_combine(seed, desc.invert_scale);
+    seed = hash_combine(seed, desc.kv_head_number);
```

Comment:
Do we also need to update this? https://github.com/oneapi-src/oneDNN/blob/f6f966988b9b0d0a3e1dcbabd4f6f7672bfb3390/src/common/serialization.cpp#L610

Reply:
And also the comparison function: https://github.com/oneapi-src/oneDNN/blob/f6f966988b9b0d0a3e1dcbabd4f6f7672bfb3390/src/common/type_helpers.hpp#L972

Reply:
Yes, Updating it now.